src/context/input_reader.rs

```diff
@@ -153,6 +153,91 @@ impl InputReader<'_, '_> {
                 value.into()
             }
             Binding::None => false.into(),
+            Binding::AnyDigital => {
```

Comment:
`get_pressed` already returns pressed keys, so I think we don't need to call things like `self.keys.any_pressed([*key])`.

I think we can simplify all this logic. Here is what I'd do:
1. Create iterators using `get_pressed` for keys, mouse_buttons, etc., with mapping their values into `Binding`. Use a separate variable for each iterator.
2. Chain all iterators, filter out all ignored values and call `next().is_some()`.